Example/Segmentify-Demo/Controller/HomeViewController.swift

```diff
@@ -33,11 +34,23 @@ class HomeViewController: UIViewController {
         // no large title
         navigationController?.navigationBar.prefersLargeTitles = false
         // pageView Request
-        sendPageViewRequest()
+        //sendPageViewRequest()
+        sendSearchPageViewRequest()
         // update table view
         tableView.reloadData()
     }
     
+    func sendSearchPageViewRequest() {
+        let pageViewObj = SearchPageModel()
+        pageViewObj.query = ""
+        pageViewObj.lang = "TR"
```

Comment:
why we set the language as "TR" .. because it renklidolap?

Reply:
@ackepenek this code is in test side of the package. I tested it with renklidolap thats why it is "TR"